src/commands/settings/scan-members.ts

```diff
@@ -0,0 +1,79 @@
+import { CommandInteraction } from 'discord.js';
+import { WriteBatch } from '@google-cloud/firestore';
+
+// TODO: GDPR/CCPA compliance (applies to entire bot, tbh)
```

Comment:
lol

Reply:
Put it in this file specifically because the code here dumps every user in a guild indiscriminately into a database. Figured might as well mention it lol

Reply:
CCPA does not apply to us. Can't speak for GDPR.